pulsar-testclient/src/main/java/org/apache/pulsar/proxy/socket/client/PerformanceClient.java

```diff
@@ -185,13 +185,11 @@ public Arguments loadArguments(String[] args) {
             arguments.proxyURL += "/";
         }
 
-        arguments.testTime = TimeUnit.SECONDS.toMillis(arguments.testTime);
```

Comment:
Why removed this? The performance consumer is using that.

Reply:
There is no need to perform unit conversion here, just pass the original value(unit is second). Because if it is converted here, we need to convert it back when printing later(code in [line251](https://github.com/apache/pulsar/pull/12020/files#diff-cf545bbba2c55f89cbaf23d8ef206e8ac765b82e2fe0652c3253a23c310f9d36R251)). And it’s not handled(that is, unit conversion) in `PerformanceConsumer.java`,`PerformanceReader.java` and `PerformanceProducer.java`.